docs/content/en/docs/releases/1.13.0.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "1.13.0"
+linkTitle: 1.13.0
+weight: 90
+---
+
+## Features
+
+To be updated
```

Comment:
1. Add ImagePullPolicy and PodSpec for KEDA connector deployments
2. RequestsPerPod and OnceOnly fields are added 
3. Websocket support for reaping 
4. Performance improvements in Router with SharedInformer